core/src/main/resources/defaults/blueprints/7.0.2/cdp-cod-edge-node.bp

```diff
@@ -0,0 +1,53 @@
+{
+  "description": "COD Edge Node cluster template",
+  "blueprint": {
+    "cdhVersion": "7.0.2",
+    "displayName": "opdb",
+    "cmVersion": null,
+    "repositories": [],
+    "products": [],
```

Comment:
These values are not defined in our original blueprint but I assume these are the default values.

Reply:
Yes, this is something I got from templates on mow-dev, so these are the default values.